mmdet/models/losses/gfocal_loss.py

```diff
@@ -0,0 +1,107 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import LOSSES
+from .utils import weight_reduce_loss
+
+
+def quality_focal_loss(pred,
+                       label,
+                       score,
+                       weight=None,
+                       beta=2.0,
+                       reduction='mean',
+                       avg_factor=None):
+    # all goes to 0
```

Comment:
Could use a bit more explanation to what this means (the formula in your paper is very intuitive)

Reply:
Thanks for advices. I would add more clear comments in this functions~